src/scenes/MerchStorePage/PurchaseBox.tsx

```diff
@@ -300,19 +354,27 @@ const StyledCheckoutBox = styled.div<{
       pointer-events: all;
     `}
 
-    // This shows the receipt text and box on mobile above the actual asset
     ${({ isReceiptState }) =>
       isReceiptState &&
-      `position: fixed;
-       top: 16px;
-       height: auto;
-       bottom: unset;
-       left: 50%;
-       transform: translateX(-50%);
-       width: min(340px, 90vw);
-       background: rgba(255,255,255,.5);
-       backdrop-filter: blur(4px);
-      `}
+      `
+      display: none;
+       `}
+`;
+
+const StyledMobileReceipt = styled.div`
+  z-index: 2;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  overflow: hidden;
+  border: 1px solid transparent;
+  padding: 16px;
+  border: 1px solid ${colors.porcelain};
+  transition: ${transitions.cubic};
+  position: relative;
+  width: min(340px, 90vw);
+  margin-top: 5vh;
 `;
```

Comment:
This is the same as old styling but without its fixed positioning. We inject this into the parent page based on state